lib/gears/timer.lua

```diff
@@ -67,6 +67,7 @@ local protected_call = require("gears.protected_call")
 -- to enable garbage collection.
 -- @tfield number timeout Interval in seconds to emit the timeout signal.
 --   Can be any value, including floating point ones (e.g. 1.5 seconds).
+--   It will use a seconds-based timeout for integer values.
```

Comment:
TODO: not really clear/complete

Reply:
mb better mention real names of glib functions